app/src/test/java/io/intrepid/contest/screens/contestcreation/reviewcontest/ReviewContestPresenterTest.java

```diff
@@ -0,0 +1,51 @@
+package io.intrepid.contest.screens.contestcreation.reviewcontest;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import io.intrepid.contest.models.Contest;
+import io.intrepid.contest.testutils.TestPresenterConfiguration;
+
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
This shouldn't be necessary. Also in general, you only need to have either the `@RunWith` annotation or the `MockitoAnnotations.initmocks(this)` method called, but you shouldn't need both. However, in this case, the `BasePresenterTest` should take care of initializing your mocks for you since it has a `MockitoRule` declared.